Assets/Scripts/Utlities/NotificationUtility.cs

```diff
@@ -156,7 +156,7 @@ private IEnumerator NotifyEnumerator(float stayTime)
                 yield return null;
             }
 
-            yield return new WaitForSeconds(stayTime);
+            yield return new WaitForSeconds(stayTime + 4f);
```

Comment:
Kan je ipv hardcoded 4f value toevoegen niet de stayTime value aanpassen?

Reply:
Kan, ik dacht ook dat dat makkelijker zou zijn. Maar stayTime is enum constructie waar verschillende waardes op verschillende plekken wordt gebruikt. Ik kan er straks nog even naar kijken. Dan zou ik het eventueel rechtstreeks in development kunnen pushen.